packages/voila/src/manager.ts

```diff
@@ -238,6 +243,12 @@ export class WidgetManager extends JupyterLabManager {
 
         // Send a states request msg
         initComm.send({ method: 'request_states' }, {});
+
+        // Reject if we didn't get a response in time
```

Comment:
Could you add `TODO` here to remove this code in the future?

Reply:
Are you referring to my comment?

> Once we reuse the base manager, we can get rid of this code in Voila.

The entire method `_build_models` will be removed once https://github.com/jupyter-widgets/ipywidgets/pull/3335 gets released.

Reply:
Thanks for clarifying!